Makefile

```diff
@@ -28,7 +28,7 @@ clean:
 	for d in lambdas terraform; do $(MAKE) -C $$d clean; done
 
 test:
-	$(MAKE) -C test
+	PYTHONWARNINGS=ignore:ResourceWarning coverage run -m unittest discover test --verbose
 
 travis:
```

Comment:
Can you remove this target while you're at it? I think it is unused.

Reply:
The travis target? Yeah, I can do that.